src/PhraseExtract.cpp

```diff
@@ -101,7 +101,7 @@ class PhraseExtract::DictType {
   void BuildTrie() {
     std::unordered_map<std::string, int> key_item_id_map;
     marisa::Keyset keyset;
-    for (int i = 0; i < items.size(); i++) {
+    for (size_t i = 0; i < items.size(); i++) {
       const auto& key = items[i].first;
       key_item_id_map[key.ToString()] = i;
```

Comment:
`key_item_id_map` has type `std::unordered_map<std::string, int>`, but `i` is `size_t`.